bundles/org.eclipse.core.resources/src/org/eclipse/core/internal/watson/ElementTreeWriter.java

```diff
@@ -79,11 +79,17 @@ public Object readData(IPath path, DataInput input) {
 	}
 
 	/**
-	 * Sorts the given array of trees and The sort order is written to the given
-	 * output stream.
+	 * Writes indexes of sorted trees to output stream.
```

Comment:
... to _an_ output stream

Reply:
It's rather "the" given argument. Lets focus on more severe issues.